sopel/db.py

```diff
@@ -109,10 +114,10 @@ class SopelDB(object):
     def __init__(self, config):
         # MySQL - mysql://username:password@localhost/db
         # SQLite - sqlite:////home/sopel/.sopel/default.db
-        db_type = config.core.db_type
+        self.type = config.core.db_type
```

Comment:
And also the config option is `db_type`, which is a fine name.

Reply:
An object attribute can be named `type`, however, without conflicting anywhere. It will always be accessed as `self.type` or `db.type` or `bot.db.type`. Writing `bot.db.db_type` looks redundant.

Reply:
Oh well, I tried.